wake/demo.wake

```diff
@@ -47,19 +50,30 @@ global def demo =
 
   makeTestProgramPlan programName cFiles
   | setTestProgramPlanIncludeDirs ("{blockPIOSifiveRoot}/metal/drivers", Nil)
+  | setTestProgramPlanCFlags ("-DPIO_BASE=0x60000", Nil)
+
+global def demo16 =
+  demo
+  | setTestProgramPlanName "demo16"
+  | setTestProgramPlanCFlags ("-DPIO_WIDTH_16", "-DPIO_BASE=0x70000", Nil)
```

Comment:
This can get deleted once we update @sistillson's PR, since that should allow the test to dynamically update based on the PIO params and address, right?

Reply:
yeah